src/components/homepage/App.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import React from 'react';
+import Header from './Header';
+import Background from './Background';
+
+class App extends React.Component {
+  render(){
+    return(
+      <div className="App">
+        <div className="Background">
+          {Background}
```

Comment:
Would `coverImg` or sth be a more appropriate name so we don't get confused later?